pom.xml

```diff
@@ -1307,8 +1310,8 @@
             <maxRank>6</maxRank>
             <effort>Max</effort>
             <xmlOutput>true</xmlOutput>
-            <failOnError>${findbugs.failOnViolation}</failOnError>
-            <excludeFilterFile>findbugs-excludes.xml</excludeFilterFile>
+	    <failOnError>${spotbugs.failOnViolation}</failOnError>
+	    <excludeFilterFile>spotbugs-excludes.xml</excludeFilterFile>
```

Comment:
@jhrcek is this correct - I mean without any path?

Reply:
so we will have the general filter and these in submodules too?